link.go

```diff
@@ -545,10 +545,12 @@ type LinkXDP struct {
 func (xdp *LinkXDP) decode(ad *netlink.AttributeDecoder) error {
 	for ad.Next() {
 		switch ad.Type() {
+		// this is some ugly code for the moment, this should be in the
```

Comment:
Why? This looks idiomatic to me.

Reply:
Seems like I didn't clean this comment when I changed from some dirty code to idiomatic one. Removed the comment.